stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -712,13 +712,13 @@ end
 ###########################################################################################
 
 """
-    rank(A[, tol::Real])
+    rank(A::AbstractArray, atol::Real, rtol::Real)
+    rank(A[, tol::Real]) = rank(A, rtol=tol)
```

Comment:
If we're going to deprecate this, should we remove it from the docs?

Reply:
It's not deprecated yet.

Reply:
Maybe then a comment that this will be deprecated in future?

Reply:
Hi @simonbyrne , It has already been[ added here](https://github.com/JuliaLang/julia/blob/d9527390420340a73a1358076291f350ffbb8a41/stdlib/LinearAlgebra/src/generic.jl#L750).

Reply:
I meant add a comment in the docs that it will be removed in future

Reply:
I meant add a comment in the docs that it will be removed in future

Reply:
Resolved :+1: . I deprecated `tol` in `rank`.